cm.py

```diff
@@ -514,3 +518,29 @@ def apply_host_template(self, cluster_name, host_template_name, start_roles, hos
                                                    host_template_name=host_template_name,
                                                    start_roles=start_roles,
                                                    host_ref_list=host_ref_list)
+
+    def first_run_service(self, cluster_name, service_name, timeout=180):
+        command_id = self.api_client.first_run_cluster_service(cluster_name=cluster_name,
+                                                               service_name=service_name)['id']
+        _run_command(self, command_id, timeout)
+
+
+def _run_command(object, command_id, timeout=180):
```

Comment:
I think it might be more accurate to call this `wait_for_command` or something similar?

Reply:
Done.